android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/WalletUtils.java

```diff
@@ -2,29 +2,29 @@
 
 import android.app.Activity;
 import android.app.AlertDialog;
-import android.content.Context;
 import android.content.DialogInterface;
 import android.content.Intent;
 import android.content.pm.PackageManager;
 import android.net.Uri;
-import android.util.ArrayMap;
 import android.util.Log;
 import com.appcoins.sdk.android.billing.BuildConfig;
-import com.appcoins.sdk.android.billing.R;
-import java.lang.reflect.Field;
+import java.lang.ref.WeakReference;
 
 public class WalletUtils {
 
+  private static final String TAG = AppcoinsBillingStubHelper.class.getSimpleName();
   public static String walletPackageName = BuildConfig.BDS_WALLET_PACKAGE_NAME;
+  public static String aptoidePackageName = BuildConfig.APTOIDE_PACKAGE_NAME;
```

Comment:
Do we need a local variable for this parameters since we are going for the BuildConfigs, since this is an utility class with static para meters this should be provided in the method call. If the idea is to have an utility class that is configured with parameters, please consider having it defined as an standard class without static methods